src/framework/core/js/DataBinding.js

```diff
@@ -578,8 +578,10 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
                 segs: sourceSegs,
                 transactional: options.transactional
             }, sourceListener);
-            fluid.log(fluid.logLevel.TRACE, "Adding relay listener with listenerId " + spec.listenerId + " to source applier with id " +
-                sourceApplier.applierId + " from target applier with id " + applierId + " for target component with id " + target.id);
+            if (fluid.passLogLevel(fluid.logLevel.TRACE)) {
```

Comment:
It seems ```fluid.log()``` function already checks the condition of ```fluid.passLogLevel``` if the first argument is a logLevel: https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/Fluid.js#L301

Is it still necessary to add the ```if``` condition around ```fluid.log()```?

Same question applies to other changes.